autoload/fakeclip.vim

```diff
@@ -25,6 +25,8 @@
 
 if has('macunix') || system('uname') =~? '^darwin'
   let s:PLATFORM = 'mac'
+elseif system('cat /proc/sys/kernel/osrelease') =~? 'Microsoft'
+  let s:PLATFORM = 'bashwin'
```

Comment:
It seems that ["bashwin" is not a proper abbreviation for Bash for Windows](https://www.google.com/search?q=%22bashwin%22%20%22bash%20on%20windows%22).  It would be better to use `wsl` instead?